components/library/openssl/openssl-1.0.2/Makefile

```diff
@@ -44,8 +45,6 @@ include $(WS_MAKE_RULES)/configure.mk
 include $(WS_MAKE_RULES)/ips.mk
 include $(WS_MAKE_RULES)/lint-libraries.mk
 
-PATH=$(GCC_ROOT)/bin:/usr/bin:/usr/gnu/bin:/usr/perl5/bin
```

Comment:
Yes, I can do that.  It worked without it because /usr/bin/gcc was a symlink to the actual compiler.